### PR TITLE
Add most verbose IF JSON payloads

### PR DESCRIPTION
**Pull request by ajchilts**

### CODE REVIEW

File: resources/public/api/conf/1.0/examples/create-contact-details-response.json [hunk 1]
```diff
@@ -1,3 +1,33 @@
 {
-  "TODO": "TODO"
+  "contactDetails": [
+    {
+      "code": 9,
+      "detail": "07123 987654",
+      "type": "MOBILE TELEPHONE"
+    },
+    {
+      "code": 8,
```
Comment (parkeradam) [inline resources/public/api/conf/1.0/examples/create-contact-details-response.json:9]: Code is unneeded here - none o fthe OGDs ask for this, same in create.

File: resources/public/api/conf/1.0/schemas/create-contact-details-response.json [hunk 2]
```diff
@@ -1,3 +1,81 @@
...
+  "description": "Addresses",
+  "type": "object",
+  "properties": {
+    "contactDetails": {
+      "type": "array",
+      "description": "The list of the individual's contact telephone numbers.",
+      "items": {
+        "type": "object",
+        "properties": {
+          "code": {
```
Comment (parkeradam) [inline resources/public/api/conf/1.0/schemas/create-contact-details-response.json:12]: Unneeded as no OGDs ask for it -> Same wioll need to be removed from create docs.

**Review by parkeradam — CHANGES_REQUESTED** [review]

File: resources/public/api/conf/1.0/examples/create-contact-details-response.json (hunk 1 above)
Comment (ajchilts) [reply resources/public/api/conf/1.0/examples/create-contact-details-response.json:9]: For some reason I thought we filtered on this but its type.

Comment (ajchilts) [reply resources/public/api/conf/1.0/examples/create-contact-details-response.json:9]: @parkeradam 

https://github.com/hmrc/individuals-details-api/blob/master/conf/application.conf#L245

https://github.com/hmrc/individuals-details-api/blob/master/conf/application.conf#L245

Looks required in Details?

**Review by parkeradam — APPROVED** [review]

**Pull request merged**